arches/settings.py

```diff
@@ -371,6 +372,7 @@
     "django.contrib.auth.middleware.AuthenticationMiddleware",
     "django.contrib.messages.middleware.MessageMiddleware",
     # "django.middleware.clickjacking.XFrameOptionsMiddleware",
+    #"dauthz.middlewares.request_middleware.RequestMiddleware",
```

Comment:
These can go, but maybe useful for any reviewers to see what plugging is required, until PR closer to ready.